core/src/test/scala/unit/kafka/admin/AdminTest.scala

```diff
@@ -95,6 +90,7 @@ class AdminTest extends ZooKeeperTestHarness with Logging with RackAwareTest {
   }
 
   @Test
+  @deprecated("This test has been deprecated and will be removed in a future release.", "1.1.0")
```

Comment:
Wouldn't it be better to deprecate the whole test class and move any non-deprecated tests elsewhere?

Reply:
Yes, thats good idea.
 I am going to move:
ReassignPartitions related test to ReassignPartitionsCommandTest
PreferrredReplica related tests to a new test class PreferredReplicaElectionCommandTest
testControlledShutdown test to ControllerIntegrationTest
testReplicaAssignment test to AdminRackAwareTest 

Let me know, If any concerns.